build-scripts/webpack.js

```diff
@@ -70,6 +70,7 @@ const createWebpackConfig = ({
         __BUILD__: JSON.stringify(latestBuild ? "latest" : "es5"),
         __VERSION__: JSON.stringify(version),
         __DEMO__: false,
+        __BACKWARDS_COMPAT__: false,
```

Comment:
This new constant allows us to add some code paths to remain backwards compatibility. This path will be removed if backwards compat is turned off. 

Currently only Cast will have this turned on.